test/unit/preprocessing/linear_dep_remover/test_sklearn_compat.py

```diff
@@ -30,7 +30,6 @@
         estimator_checks.check_dtype_object,
         estimator_checks.check_sample_weights_pandas_series,
         estimator_checks.check_sample_weights_list,
-        estimator_checks.check_sample_weights_invariance,
```

Comment:
not sure why this needs to be removed here.

Reply:
Ah I saw your commit message. Fair enough (I would have made it a separate PR, but happy to have it here)

Reply:
@riedgar-ms is tackling this in #669 which is very likely going in before this, so it'll be removed in the next sync. I just wanted to see if the builds pass.

Reply:
#669 is completed and this is now gone from this PR. Closing